This patch will allow the browser to load in low memory condition.s
i.e. it'll allow the palm web browser to come up in sitatuions when you would normally see the 'too many cards' error.

11/17/09
diomark / precentral
Index: /etc/palm/luna.conf
===================================================================
--- .orig/etc/palm/luna.conf
+++ /etc/palm/luna.conf
@@ -52,7 +52,7 @@ SwapRateThreshold=200
 CardLimit=-1
 EnableMemoryTracking=true
 UseMemchute=true
-AppsToAllowInLowMemory=com.palm.app.phone;com.palm.app.contacts;com.palm.app.messaging
+AppsToAllowInLowMemory=com.palm.app.phone;com.palm.app.contacts;com.palm.app.messaging;com.palm.app.browser
 
 [TouchEvents]
 TapRadiusSquared=576
